src/parse_stream.jl

```diff
@@ -70,7 +81,11 @@ function untokenize(head::SyntaxHead; unique=true, include_flag_suff=true)
     if include_flag_suff && suffix_flags != EMPTY_FLAGS
         str = str*"-"
         is_trivia(head)  && (str = str*"t")
-        is_infix(head)   && (str = str*"i")
+        is_infix_op_call(head)          && (str = str*"i")
+        # call op flag name mnemonic: i-infix, h,j - left and right of `i` in
+        # alphabetic order
+        is_prefix_op_call(head) && (str = str*"h")
+        is_suffix_op_call(head) && (str = str*"j")
```

Comment:
We're not limited to one-char suffixes here, right? So `-pre`/`-suf` or `-pre`/`-post` would probably be the more obvious choices.

Reply:
Flags can be combined (eg raw and string), so I did limit it to one char suffixes on purpose.

Could be a good idea though to make these clearer. But if we do, let's do all the flags at once.

Reply:
(ie, in a separate PR)

Reply:
Actually I'll do this now after all: These particular flags aren't combined with others so this will be clearer and the lack of consistency shouldn't be a big deal.

Reply:
Done